resources/entrypoints/fix_user.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+echo "[Entrypoint] RonDB Docker Image"
+
+echo "\$@: $@"
+
+# In the case we're running on Linux and the user executing build_run_docker.sh
+# is not 1000:1000, we will fail writing files to the host file system. We thereby
+# add the mysql user to the group of host user.
+
+# We need to be in the same group as the host to be able to create files
```

Comment:
I know you abandoned this in a later commit, but if you don't mind I could take a stab at it, so that both CI and other users can run with local directories and also be able to read/delete files. Do you know anything that would prevent that?

Reply:
Well yes, you can take a stab at it. I personally just got worried that it might reach a level of hackiness where others may not understand what is happening anymore. Me included, in a couple of months.

I'd prefer to create a workaround there if we actually get that feedback from outsiders who can't run this.

Reply:
Then I'll take a stab at it. I think the `--gid` workaround could be a workaround the user doesn't have to do anything in order to use, and we also retain the `-lv` functionality.

Reply:
Sure. You can also automate a GID check in the script, without requiring the user to supply it.

Reply:
I did add a commit there somewhere where I checked that. Also checked whether `uname -s` == "Darwin". Could look that up, I also removed that later on.

Reply:
Thanks, I now have a working solution that uses your code.